docs/Installation.rst

```diff
@@ -11,137 +11,22 @@ The tool has two different components(Backend and Frontend). Each of them has it
-
-``django-admin --version``
-
-The output is something like: ``2.2.2``.
-
-
-**Create the phabricator account and generate a Conduit API token:**
-
-Before running the development server, you need to provide an API key to fetch the details from the phabricator. So create a phabricator account from `here <https://phabricator.wikimedia.org/auth/start/?next=%2F>`_. Use **Log In or Register** through Mediawiki(If you don't have a Mediawiki account, you can create it from `here <https://www.mediawiki.org/w/index.php?title=Special:CreateAccount>`_.
-
-Once you login to the phabricator. You can generate a Conduit API token from ``https://phabricator.wikimedia.org/settings/user/{Your username}/page/apitokens/`` (fill your username in the link).
-
-Copy the API token, and paste it in the variable named ``API_TOKEN`` in the file ``backend/WikiContrib/WikiContrib/settings.py``
-
-**Run the migrations:**
-
-Now, you need to run the migration files. Type the following command:
-
-``python manage.py migrate``
-
-**Create super user:**
-
-You have successfully created the schema now. To access the models you need to create a superuser. Use this command:
-
-``python manage.py createsuperuser``
-
-The above prompts for username, email, and password. The command creates a user with the corresponding username and password. You can see the database tables or models using it.
-
-
-**Run the local server:**
-
-Finally, run the server with the command:
-
-``python manage.py runserver``
-
-Hurray!! Now you can access the API. You can get the API doc `here <https://documenter.getpostman.com/view/6222710/SVYurxMj?version=latest>`_.
-
-To see the database tables, you can go to this url: ``http://127.0.0.1:8000/admin/`` and login with the above created credentials.
+cd into ``WikiContrib`` and follow the instructions `here <https://github.com/wikimedia/WikiContrib/blob/master/backend/WikiContrib/Install.md/>`_ to install the backend .
 
 
 
 Frontend
 ========
 
-If you go inside ``frontend`` directory( use command ``cd frontend``). You can find another directory named ``WikiContrib-Frontend``. It is the main project directory.
-
-**Steps to setup server locally**
-
-1. Install npm
-2. Install the requirements.
-3. Start the development server.
-
-**Installing npm**
-
-``sudo apt-get update``
-
-``sudo apt-get install nodejs``
-
-``sudo apt-get install npm``
-
-You have installed npm successfully. You can check the version of npm with the command ``npm -v``
-
-**Install the requirements.**
-
-Now Inside the directory ``WikiContrib-Frontend``, type the command
-
-``npm install``
-
-This installs all the requirements to the tool.
-
-**Start the development server.**
-
-Now type the following command in the same directory.
-
-``npm start``
-
-This starts a development server in a URL like ``http://localhost:3000/``. Hurray! you have successfully hosted the fronend in local environment.
-
+from the projects root directory, there will be two directories in it:
```

Comment:
Replace the sentence with "You will see two directories in the project's root directory:"